index.html

```diff
@@ -192,6 +192,42 @@ <h1 class="about-title">About me</h1>
 
   </section>
 
+<section class="contact-page">
+  <h2 class="contact-comment">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+  <form action="https://formspree.io/f/mqkoaqdk/" method="POST">
+
+      <ul class="form-list">
```

Comment:
[OPTIONAL] Kindly add some more space before the form input areas start like on the Figma template. We need to code the Figma design pixel perfectly as much as possible 👍 
<img width="515" alt="Screenshot 2023-03-28 at 9 37 08 PM" src="https://user-images.githubusercontent.com/25887253/228337104-3bc31a8c-284e-4cb5-8f20-11569e27b635.png">
<img width="515" alt="Screenshot 2023-03-28 at 9 37 17 PM" src="https://user-images.githubusercontent.com/25887253/228337113-6847065c-c4e9-479b-86a3-a37e4dfbe210.png">